app/topics/show-topic/show-topic.component.html

```diff
@@ -55,10 +55,17 @@
     <div class="title-wrapper">
       {{ 'subtopics' | translate }}
     </div>
-    <button md-fab class="add-topic">
-      <md-icon class="md-fab add-icon">add</md-icon>
-    </button>
   </md-card-title>
+  <div class="add-subtopic-button">
+    <button class="md-fab add-new-subtopic-button" (click)="addSubtopic()" [routerLink]="[ '/topics/', topic.id, 'new-subtopic']">
+      <md-icon class="md-24 add-icon">add</md-icon>
+    </button>
+    <button class="md-fab add-existing-topic-button" (click)= "addFromExitingTopic()">
```

Comment:
I would put this in the same button. Then we can distinguish  between existing and net topic